client/src/redux/reducers/workbench.reducers.js

```diff
@@ -2,8 +2,7 @@ export const initialState = {
   loading: true,
   config: {},
   portalName: '',
-  setupComplete: window.__INITIAL_SETUP_COMPLETE__,
-  hideDataFiles: false
```

Comment:
`hideDataFiles` is under `config`.  This change is unrelated to PR but something I noticed along the way.